src/components/NewWorkspaceModal.js

```diff
@@ -91,7 +91,7 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
       onSuccess(await Utils.cond(
         [cloneWorkspace, async () => {
           const workspace = await Ajax().Workspaces.workspace(cloneWorkspace.workspace.namespace, cloneWorkspace.workspace.name).clone(body)
-          const featuredList = await Ajax().Buckets.getFeaturedWorkspaces()
+          const featuredList = await Ajax().FirecloudBucket.getFeaturedWorkspaces()
```

Comment:
I chose `FirecloudBucket` because the config that is being referenced is `firecloudBucketRoot`. I agree that better names would be good, but for now consistency with the config seemed like my best option.